src/core_plugins/kibana/public/context/query/actions.js

```diff
@@ -84,7 +84,17 @@ export function QueryActionsProvider(courier, Notifier, Private, Promise) {
     setLoadingStatus(state)('predecessors');
 
     return Promise.try(() => (
-      fetchPredecessors(indexPatternId, anchor, [_.zipObject([sort]), { [tieBreakerField]: 'asc' }], predecessorCount, filters)
+      fetchPredecessors(
+        indexPatternId,
+        sort[0],
+        sort[1],
+        anchor.sort[0],
+        tieBreakerField,
+        'asc',
+        anchor.sort[1],
```

Comment:
Just a suggestion, but I always find numerical index based access hard to read. This might be a bit easier to understand if you declare these as variables with names. Especially since (I think) `sort` and `anchor.sort` contain two different types of data (`sort` being a query sort object and `anchor.sort` being the sort values anchor contained, right?).

Reply:
I agree with the readability issue. The `sort` value is passed from discover and re-used in the doc-table, which is why I maintained its structure. `anchor.sort` is whatever Elasticsearch returns as sort values from the anchor query. I will investigate improving both. :+1: